File: tesseract-ocr-rest/src/main/java/com/swayam/ocr/porua/tesseract/model/PageImage.java
package com.swayam.ocr.porua.tesseract.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.OneToOne;
import javax.persistence.Table;

import lombok.Data;

@Entity
@Table(name = "page_image")
@Data
public class PageImage {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;

    @OneToOne(fetch = FetchType.EAGER)
    @JoinColumn(name = "book_id")
    private Book book;

    @Column
    private String name;

    @Column(name = "page_number")
    private int pageNumber;

}

File: ocr-correction-rest/com/swayam/ocr/porua/model/PageImage.php
<?php

namespace com\swayam\ocr\porua\model;

/**
 * @Entity
 * @Table(name="page_image")
 */
class PageImage {

    /**
     * @Id
     * @Column(type="bigint")
     * @GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @OneToOne(targetEntity="Book")
     * @JoinColumn(name="book_id", referencedColumnName="id")
     */
    private $book;

    /** @Column */
    private $name;

    /** @Column(name = "page_number", type="integer") */
    private $pageNumber;

    /** @Column(type="boolean") */
    private $ignored;

    /** @Column(name = "correction_completed", type="boolean") */
    private $correctionCompleted;

}
